ui/pages/reservation-unit/[id].tsx

```diff
@@ -1197,31 +1156,26 @@ const ReservationUnit = ({
                 <PaddedContent>
                   {futurePricing && (
                     <p style={{ marginTop: 0 }}>
-                      <Trans i18nKey="reservationUnit:futurePricingNotice">
-                        Huomioi{" "}
+                      <Trans
+                        defaults="Huomioi <bold>hinnoittelumuutos {{date}} alkaen. Uusi hinta on {{price}}</bold>."
+                        values={{
+                          date: toUIDate(new Date(futurePricing.begins)),
+                          price: getPrice({
+                            pricing: futurePricing,
+                            trailingZeros: true,
+                          }).toLocaleLowerCase(),
+                        }}
+                        components={{ bold: <strong /> }}
+                      />
```

Comment:
Missing i18nKey. Need to refactor the translation tags.
See https://react.i18next.com/latest/trans-component#alternative-usage-which-lists-the-components-v11.6.0